### ISSUE TITLE
docComments removes macro documentation

### ISSUE THREAD

**Issue by Finnvoor** [nicklockwood/SwiftFormat#1552]
The following top level declaration:

```swift
/// Adds a static `logger` member to the type.
/// - Parameters:
///   - subsystem: An optional subsystem for the logger to use. Defaults to `Bundle.main.bundleIdentifier`.
///   - category: An optional category for the logger to use. Defaults to `String(describing: Self.self)`.
@attached(member, names: named(logger)) public macro StaticLogger(
    subsystem: String? = nil,
    category: String? = nil
) = #externalMacro(module: "StaticLoggerMacros", type: "StaticLogger")
```

gets formatted to:
(doc comments converted to regular comments)

```swift
// Adds a static `logger` member to the type.
// - Parameters:
//   - subsystem: An optional subsystem for the logger to use. Defaults to `Bundle.main.bundleIdentifier`.
//   - category: An optional category for the logger to use. Defaults to `String(describing: Self.self)`.
@attached(member, names: named(logger)) public macro StaticLogger(
    subsystem: String? = nil,
    category: String? = nil
) = #externalMacro(module: "StaticLoggerMacros", type: "StaticLogger")
```

**Comment by nicklockwood** [comment]
@Finnvoor fixed in 0.52.8

**Comment by Finnvoor** [comment]
amazing, thanks 🙏
